packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -98,7 +98,7 @@ function UncontrolledInnerBlocks( props ) {
 
 	const { allowSizingOnChildren = false } = defaultLayoutBlockSupport;
 
-	const defaultLayout = useSetting( 'layout' ) || EMPTY_OBJECT;
+	const [ defaultLayout ] = useSettings( 'layout' );
```

Comment:
That's not needed because `defaultLayout` is used only with an object spread operator, `...defaultLayout`, which treats `undefined` or `null` as an empty object already.